source/features/prevent-pr-commit-link-loss.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import AlertIcon from 'octicon/alert.svg';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '../libs/features';
+import {prCommitRegex, preventPrCommitLinkBreak} from '../libs/utils';
+
+function handleButtonClick(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	const field = event.delegateTarget.form!.querySelector('textarea')!;
+	textFieldEdit.set(field, preventPrCommitLinkBreak(field.value));
```

Comment:
I implemented https://github.com/fregante/text-field-edit/pull/7

I think it can be added with: 
```sh
npm i text-field-edit@latest
```

```suggestion
	textFieldEdit.replace(field, prCommitRegex, url => `[${url} ](${url})`);
```